mantidproject/mantidimaging
# Gracefully handle mismatched stack and 180 degree shapes

Add some nice handling for the selected stack and 180 degree projection of that same stack containing different shapes when using the reconstruction window.

Fixes #729 

To test:
- Open Imaging
- Load some data without any 180 degree projections
- Crop one of the stacks to a small size (Default should be fine)
- Load a 180 degree projection for that stack that is a different shape in pixel size (height and width etc)
- Open the recon window (there should be a pop up speaking about the error being possible later that lets you proceed)
- Click "Correlate 0 and 180" which should fail "gracefully" and let you know about the problem and give a suggestion on how to fix it.
